repo.go

```diff
@@ -476,6 +514,122 @@ func (r *Repo) RevokeKeyWithExpires(keyRole, id string, expires time.Time) error
 	return err
 }
 
+// AddTargetsDelegation is equivalent to AddTargetsDelegationWithExpires, but
+// with a default expiration time.
+func (r *Repo) AddTargetsDelegation(delegator string, role data.DelegatedRole, keys []*data.PublicKey) error {
+	return r.AddTargetsDelegationWithExpires(delegator, role, keys, data.DefaultExpires("targets"))
```

Comment:
I think it might be nice to have a more compact set of APIs for manipulating targets, using the options pattern to customize expiration, preferred delegated target role, etc. This is out of scope for this PR, though.